modules/axhal/src/arch/aarch64/hv/exception.rs

```diff
@@ -1,51 +1,29 @@
 use core::arch::global_asm;
-use core::sync::atomic::{AtomicUsize, Ordering};
-
-use aarch64_cpu::registers::{ESR_EL2, FAR_EL2};
 use tock_registers::interfaces::*;
 
-use memory_addr::VirtAddr;
-use page_table_entry::MappingFlags;
-
-use super::exception_utils::*;
+global_asm!(include_str!("exception.S"));
 use crate::arch::TrapFrame;
 
-global_asm!(include_str!("exception.S"));
+use core::sync::atomic::{AtomicUsize, Ordering};
```

Comment:
move this to line 2
Dependencies from `alloc` or `core` should be placed at the top.

My preferred dependency order: Rust core/alloc dependencies, third-party independent dependencies, 
arceos-related component dependencies, and current crate dependencies. 

Separate them with blank line to prevent automatic 'cargo fmt' reformatting finally.